src/Hidden/HiddenJs.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+/**
+ * Responsively hides children by omission.
+ */
+import React, { Element } from 'react';
+import { keys as breakpoints } from '../styles/breakpoints';
```

Comment:
We could, but it would make no difference because the implementation is tightly bound to the given props which are derivatives of `'xs' | 'sm' | 'md' | 'lg' | 'xl'`.  So extra code to use theme from context provides no greater capabilities because the `props` don't recognize anything else.  To make that useful we would have to have dynamic props.